content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. It includes more than 12 new features and enhancements to help you spend more time in flow and less time manually grokking your codebase. 
+
+Our code intelligence platform semantically indexes and analyzes code and its underlying structure to help you derive knowledge from the codebase. It provides devs and engineering leaders with a complete understanding of the codebase to make faster and better decisions. Our platform enables anyone who creates or interacts with code to save time by helping you understand, fix, and automate changes across your entire codebase.
+
+Read on to discover more details about the improvement with Sourcegraph 4.0.  
+
+# 🌅 Code intelligence platform for everyday use
+
+<Badge text="Code Search" color="cerise" size="small" onClick={() => window.open("/code-search", "_self")} />
+
+### A faster, simpler search experience 
+The search interface has been streamlined and simplified to help make the platform more intuitive. It’s easier than ever for you to get started with real data on the homepage from your instance including repository names, file names, and authors. These details, along with contextual tips and recommendations, help you uncover necessary information to run a successful search query, faster. 
+
+Search results are now front and center with the improved interface. The search sidebar has moved to the right of the screen and several non-essential elements have been removed to ensure the information you’re looking for is easy to access. Filters make it easier and faster for you to extract the answer you need, and the side bar can be collapsed altogether to reduce noise. 
+
+Performance across the platform saw a boost and search results are now returned faster than before by using the latest in React, lazy loading, and priority rendering. 
```

Comment:
@lcolston08 - what type of performance boost? Faster search results? Load time? Please add specifics and use active voice.

Reply:
Thanks - This is still unclear and I don't have the answer yet. Maria is unaware and so is Eugenia. It's unclear with Ben's departure.